Darkly/EventSource/EventSource.h

```diff
@@ -0,0 +1,100 @@
+//
```

Comment:
We had to make changes to the library to support custom headers for Authentication, and if we use the pod we would have to replace those changes each time.